### PR TITLE
Bump package versions for NodeJS

### PR DESCRIPTION
**Pull request by mwest1066**
@nwalters512 what happens with building new versions of packages that have had their dependencies upgraded? Does that happen automatically or do I need to do something to trigger it?

### CODE REVIEW

File: package.json
Comment (nwalters512) [inline package.json:58]: 👀 is this a new dep?

**Review by nwalters512 — APPROVED** [review]
LGTM! If you want to release new versions of packages, run `make changeset` and follow the prompts. Skimming the packages, I think `prettier-plugin-sql` and `opentelemetry` are the only ones for which we should publish new versions.

Before merging, could you run `yarn dedupe` and verify that `yarn.lock` doesn't change?

File: package.json
Comment (jonatanschroeder) [reply package.json:58]: Seems like an old dependency removed in #7284 . Should probably be removed here.
```suggestion
```

Comment (mwest1066) [reply package.json]: Good catch, thanks, applied.

**Comment by mwest1066** [comment]
> Before merging, could you run `yarn dedupe` and verify that `yarn.lock` doesn't change?

Done in https://github.com/PrairieLearn/PrairieLearn/pull/7466/commits/2e9d7ccd7e636e89498b8b7028e5fa311e3dce31 (there were some changes).

**Pull request merged**
